src/pages/Home.jsx

```diff
@@ -51,17 +51,9 @@ function Home() {
   }, [countries, searchInput, selectedRegion]);
 
   useEffect(() => {
-    fetch(API_URL)
-      .then((response) => response.json())
-      .then((data) => {
-        setCountries(data);
-      })
-      .then(setLoading(true))
-      .catch((err) => {
-        console.error(err);
-        setLoading(false);
-      });
-  }, []);
+    fetchDate();
+    setTimeout(() => setLoading(true), 1000);
```

Comment:
What a fuck, man? You're killing me. Why, why do you wanna update loading with a timeout? Isn't there a better solution?

Reply:
I also thought it was stupid, but I couldn’t think of a good solution. Look now, I don’t use useMemo  and don’t quite understand why I need it. I can put it in CountriesDataContext and leave only the counter in the Home component, but what will that give us